code/modules/vehicle/vehicle.dm

```diff
@@ -146,8 +150,10 @@
 	if(has_gravity(src))
 		return 1
 
-	if(pulledby && pulledby != buckled_mob)	// no pulling the vehicle you're driving through space!
-		return 1
+	if(pulledby)
```

Comment:
I don't think this works correctly.

Reply:
suggestions?

Reply:
What should happen:
If you are pulled by someone, and you are not buckled to this person that is pulling you, then you can make a move of your own volition in space

What this code does:
If you are pulled by someone, then if there is someone who is buckled to you that is *not* pulling you, then you can move freely in space, regardless of whether the puller is buckled to you or not

So, instead, you want the following:
```
if(pulledby && !(pulledby in buckled_mobs))
  return TRUE
```